py/jupyterlite/pyproject.toml

```diff
@@ -8,7 +8,7 @@ authors = [
     {name = "JupyterLite Contributors"},
 ]
 dependencies = [
-    "jupyterlite-core",
+    "jupyterlite-core >=0.1.2,<0.2.0",
     "jupyterlite-pyodide-kernel >=0.0.4",
```

Comment:
I think as we also happen to know this kernel would need to be updated to `>=0.1.1` for `DriveFS`, probably worth doing here. I don't think the JS kernel has a similar requirement, so is probably fine.

Reply:
Wondering if it's still actually relevant to include the Pyodide and JS kernels by default. Initially they were included to avoid disrupting workflows too much, but now that it's been a couple of months and there is a migration guide, we can also consider removing them.

The `jupyterlite` metapackage is still useful if we want to include other packages like `ipywidgets` for example, as discussed in https://github.com/jupyterlite/jupyterlite/issues/1003.

Reply:
Sure, `0.2.0` is a good opportunity to do this, as it would break the downstream dependency cycle. This would allow, for example, merging the `jupyterlite` and `jupyterlite-core` feedstocks on `conda-forge`... no harm in the JS kernel getting shipped from there.

Reply:
> ipywidgets for example

Well, strictly speaking, it would just be `jupyterlab_widgets`, no? Probably really only interested in the labextension side of the house, weird as that is.

Of course, the Real Deal would be to finally do the labextension (or widget) that visually built sites. Still a few missing pieces to self-hosting this in a browser-side kernel, but as a bootstrapping environment, wouldn't be awful.

Reply:
> Well, strictly speaking, it would just be `jupyterlab_widgets`, no?

Yes I meant that package :+1: